patch_xformers-0.0.23.rocm.sh

```diff
@@ -5,17 +5,17 @@ export XFORMERS_FMHA_COMMON_PATH=$(python -c 'from xformers import ops as xops;
 echo $XFORMERS_FMHA_FLASH_PATH
 echo $XFORMERS_FMHA_COMMON_PATH
 
-if ! patch -R -p0 -s -f --dry-run $XFORMERS_FMHA_FLASH_PATH "./rocm_patch/flashpy_xformers-0.0.22.post7.rocm.patch"; then
+if ! patch -R -p0 -s -f --dry-run $XFORMERS_FMHA_FLASH_PATH "./rocm_patch/flashpy_xformers-0.0.23.rocm.patch"; then
```

Comment:
Can we make the version as a variable like `XFORMERS_VERSION=0.0.23`? This will make such a modification less error-prone.

Reply:
Should we retain the support for xformers version `0.0.22.post7`?

Reply:
Changes has been addressed.